torch_glow/tests/nodes/min_test.py

```diff
@@ -0,0 +1,10 @@
+import torch
+
+from tests.utils import jitVsGlow
+
+
+def test_elementwise_min():
+    def test_f(a, b):
+        return torch.min(a + a, b + b)
```

Comment:
@jackm321 I had to do these random pointless additions to get this function glow-ified... do you know why that would be the case?  I basically copied the idea from your transpose diff :)

Reply:
@bertmaher yeah so it's basically because we are making use of PTs graph fusion stuff to do the lowering from subgraphs of PT nodes to Glow and that thing needs at least two PT nodes to fuse together to create a `glow::CompilationGroup` node.
I was thinking it might be good to document this somewhere, maybe I'll throw it in pytorch.md

Reply:
Ahh, that makes sense, I half-wondered if that's what was going on.  Thanks!